pkg/asset/machines/master.go

```diff
@@ -238,7 +238,12 @@ func (m *Master) Generate(dependencies asset.Parents) error {
 
 		imageName, _ := rhcosutils.GenerateOpenStackImageName(string(*rhcosImage), clusterID.InfraID)
 
-		machines, err = openstack.Machines(clusterID.InfraID, ic, pool, imageName, "master", "master-user-data-managed")
+		trunkSupport, err := openstack.CheckNetworkExtensionAvailability(ic.Platform.OpenStack.Cloud, "trunk")
```

Comment:
What was the reason to move `CheckNetworkExtensionAvailability` function call here and add `trunkSupport` parameter to `Machines`?

Reply:
By moving the call to `checkNetworkExtensionAvailability` outside of `func Machine`, we can now unit-test `func Machines` without any call to the cloud.

Reply:
ah, yes... it's okay then :)